src/firefly/js/ui/DropDownContainer.css

```diff
@@ -28,6 +28,12 @@
     width: 100%;
 }
 
+.DD-ToolBar__version {
+    color: rgba(256,256,256,0.1);
```

Comment:
Currently, the version is virtually impossible to read, even when selected. I am wondering if it's possible to make it more readable with a lighter color, at least on mouse over.